### PR TITLE
Feature/us 018 post operation new user

### PR DESCRIPTION
**Pull request by AlejandraBC**

### CODE REVIEW

File: src/main/java/com/agilethought/atsceapi/controller/UserController.java [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+import com.agilethought.atsceapi.service.UserService;
+
+@RestController
 public class UserController {
 
+	@Autowired
+	private UserService userService;
+
+	@ResponseStatus(HttpStatus.CREATED)
+	@PostMapping(value = "/at-sce-api/api/v1/users", consumes = "application/json", produces = "application/json")
```
Comment (ivanflores15) [inline src/main/java/com/agilethought/atsceapi/controller/UserController.java]: move this as a parameter of base path API into `@RequestMapping` anotation below `@RestController` annotation
keep `/users` as part of `@PostMapping`

File: src/main/java/com/agilethought/atsceapi/domain/CreateUserRequest.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.agilethought.atsceapi.domain;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class CreateUserRequest {
```
Comment (ivanflores15) [inline src/main/java/com/agilethought/atsceapi/domain/CreateUserRequest.java]: Name of class should be _nouns_ and don't include _verbs_ ("Create")

File: src/main/java/com/agilethought/atsceapi/domain/CreateUserResponse.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package com.agilethought.atsceapi.domain;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class CreateUserResponse {
```
Comment (ivanflores15) [inline src/main/java/com/agilethought/atsceapi/domain/CreateUserResponse.java]: Name class should be nouns

**Review by ivanflores15 — CHANGES_REQUESTED** [review]
looks good once you change this. thank you.

**Review by luisgalindoluna — CHANGES_REQUESTED** [review]
Did you forget to merge from develop into your branch? 
Your controller, repo, service and serviceImpl are missing several methods that should be in there

**Pull request closed without merging**